app/components/Activity/Transaction/Transaction.js

```diff
@@ -29,8 +29,11 @@ const Transaction = ({ transaction, showActivityModal, currencyName, intl }) =>
         data-hint={intl.formatMessage({ ...messages.type })}
       >
         <Text mb={1}>
-          <FormattedMessage {...messages[type]} />
-          {transaction.closeType && <Text color="gray">{transaction.closeType}</Text>}
+          {!transaction.closeType ? (
```

Comment:
Consider removing `!` and switching components

Reply:
Updated.

There is actually a lint rule that I usually use to enforce no negatives in conditionals as it improves readability a lot IMO.

See https://eslint.org/docs/rules/no-negated-condition

I've added to the list in https://github.com/LN-Zap/zap-desktop/issues/1559